setup/install/providers/kubernetes-v2/aws-eks.md

```diff
@@ -113,55 +101,40 @@ users:
 
 ```
 
-* Execute the following commands, which will create the necessary services accounts and cluster role bindings
+(Optional) To have the Heptio authenticator assume a role to perform cluster operations (instead of the default AWS credential provider chain), uncomment the `-r` and `<role-arn>` lines and substitute an IAM role ARN to use with your user.
 
-```
-CONTEXT=aws
-kubectl create namespace spinnaker
-#kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/eks-admin-service-account.yaml 
-#kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/eks-admin-cluster-role-binding.yaml 
-kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/spinnaker-service-account.yaml 
-kubectl apply -f https://raw.githubusercontent.com/spinnaker/spinnaker.github.io/master/setup/install/providers/aws/eks/spinnaker-cluster-role-binding.yaml 
-TOKEN=$(kubectl get secret --context $CONTEXT \
-   $(kubectl get serviceaccount spinnaker-service-account \
-       --context $CONTEXT \
-       -n spinnaker \
-       -o jsonpath='{.secrets[0].name}') \
-   -n spinnaker \
-   -o jsonpath='{.data.token}' | base64 --decode)
-kubectl config set-credentials ${CONTEXT}-token-user --token $TOKEN
-kubectl config set-context $CONTEXT --user ${CONTEXT}-token-user
-```
+(Optional) To have the Heptio authenticator always use a specific named AWS credential profile (instead of the default AWS credential provider chain), uncomment the env lines and substitute `<aws-profile>` with the profile name to use.
```

Comment:
Can these two optional steps come *after* the next step (Create the necessary service accounts..)?

Reply:
As these relates to the comments in the yaml file, these should come here as optional IMO.